### DISCUSSION TITLE
Exceptions when `app/Http/Livewire` folder is missing.

### DISCUSSION THREAD

**Discussion by chris97pl** [livewire/livewire#3820] [7. Bugs (archived)] [answered]
I don't have the above folder, as all of my components are in a custom structure and registered using `LiveWire::component()`. Looks like a bug, as it should not assume anything unless required.

The exception:
`Symfony\Component\Finder\Exception\DirectoryNotFoundException`
`The "/var/www/html/app/Http/Livewire/" directory does not exist.`

**Comment by joshhanley** [comment, marked as answer]
@chris97pl it will only throw that error if it couldn't find your registered component. Livewire will check through all manually registered components and aliases first before checking the default directory `app/Http/Livewire`.

I'd double check your components are registered property and you are calling them correctly, then you shouldn't see that error.

Hope this helps!
You can also try doing a `php artisan livewire:discover` to get it to rebuild
